docs/en/pid-eaa-issuance.rst

```diff
@@ -444,6 +458,12 @@ The JWT payload is given by the following parameters:
     * - **redirect_uri**
       -  Redirection URI to which the response is intended to be sent. It MUST be an universal or app link registered with the local operating system, so this latter will provide the response to the Wallet Instance.
       - See `OpenID.Core#AuthRequest <https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest>`_.
+    * - **OAuth-Client-Attestation**
+      - It MUST be set as in the :ref:`Table of the HTTP parameters <table_http_request_claim>`.
+      - See :ref:`Table of the HTTP parameters <table_http_request_claim>`.
+    * - **OAuth-Client-Attestation-PoP**
+      - It MUST be set as in the :ref:`Table of the HTTP parameters <table_http_request_claim>`.
+      - See :ref:`Table of the HTTP parameters <table_http_request_claim>`.
```

Comment:
```suggestion
```

Reply:
see my previous comment, it confirms @m-basili 's review and asks a specific table about the http request header parameters to be included within the request